app/controllers/users_controller.rb

```diff
@@ -81,6 +80,13 @@ def conferences
     end
     helper_method :conferences
 
+    def teams
+      all_teams = Team.all
+      active_teams = Team.where(season: Season.current, kind: %w(sponsored voluntary)).order(:name)
```

Comment:
Wouldn't it be better to move the active_teams query to the teams model? Especially because there already lives a :by_season_phase scope. https://github.com/rails-girls-summer-of-code/rgsoc-teams/blob/master/app/models/team.rb#L49
I think I can't use that as is, because of the team visibility thingy in there.